test/OffsetHelper.test.ts

```diff
@@ -92,17 +88,21 @@ describe("Offset Helper - autoOffset", function () {
       })
     );
 
+    await IWETH__factory.connect(addresses.wmatic, addr2).deposit({
```

Comment:
why not call deposit from the `wmatic` variable that was instantiated above?

Reply:
Because the variable will not be defined here yet, this is in a `before` hook that is executed before `beforeEach`. That's how mocha works...

Speaking of, I also wanted to propose to use proper fixtures in the test (`loadFixture`), much less error prone than initiating variables in `beforeEach` and avoids side effects. See https://hardhat.org/hardhat-runner/docs/guides/test-contracts#using-fixtures

Reply:
This `loadFixture` thing is really cool. I didn't know about it. It'd be lovely if you opened an issue for it 🙏🏻 It could also be a `good first issue` to encourage other devs to do an open-source contribution (definitely something I want to see more of 💚)

Reply:
Opened #5 I cannot set labels though ;)